src/main/io/osd.c

```diff
@@ -761,7 +755,7 @@ void pgResetFn_osdConfig(osdConfig_t *osdConfig)
 static void osdDrawLogo(int x, int y)
 {
     // display logo and help
-    char fontOffset = 160;
+    unsigned char fontOffset = 160;
```

Comment:
Yes, this is one of the reasons I want to get the extended memory in the AB7456 working then we can share "standard" fonts with MWOSD while having "extended" fonts for Betaflight specific stuff.